lib/rubygems/commands/update_command.rb

```diff
@@ -306,4 +315,10 @@ def which_to_update(highest_installed_gems, gem_names, system = false)
     result
   end
 
+  private
+
+  def oldest_supported_version
+    @oldest_supported_version ||= Gem::Version.new("2.5.2")
```

Comment:
What's the rationale behind deciding to support `>= 2.5.2` for now?

Reply:
It's the rubygems version that comes by default with the oldest ruby that we support (2.3).

Reply:
I'm just thinking about the way to not forget about this to move forward once 2.3 support would be dropped.

Reply:
maybe some kind of `Ruby 2.3` related comment would be enough

Reply:
Aha. I'll add a more useful test that will break if we forget to do that :+1:.